### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by JoaoCarabetta** [msfernandes/pycamara#1]
Eai,

ficou pronto o wrapper?

Tem alguma documentação?

abraço

**Comment by msfernandes** [comment]
Opa! E aí @JoaoCarabetta !
Então... Ainda não está pronto e eu parei um pouco o desenvolvimento porque a nova API dos dados abertos não tem os dados que eu preciso... Vou retomar assim que eu entregar meu TCC :)

Estou usando um wrapper que eu desenvolvi em outra organização (https://github.com/tenhodito/pygov-br/tree/dev), que também não está terminada. Está faltando terminar somente a parte do Django, o consumo do webservice já tratando o formato dos dados está pronto (na branch dev).

Quanto à documentação, na biblioteca que mencionei acima (pygov-br) o código está bem documentado, com exemplos de output e tudo mais... No pycamara eu ia montar uma documentação bacana quando terminasse...

**Comment by JoaoCarabetta** [comment]
Massa, 

vi que o pygov tem o objetivo de juntar as APIs de dados do governo federal. Eu também comecei a fazer um wrapper desse, https://github.com/labFGV/bradata. Seria interessante se a gente conseguisse trabalhar juntos nisso, não?

O @odanoburu, @joaomeirelles também tem vários projetos nesse sentido e tem interesse em contruir uma plataforma dessas.

Sobre os dados da câmara, qual deles você precisa?

**Comment by msfernandes** [comment]
@JoaoCarabetta Claro! Ia ser muito interessante se a gente fizesse isso jutnos, sim!
Vamos fazer sim, super apoio!

Eu precisava dos dados de Discurso dos deputados, mas não era prioridade pra API nova da Camara, então só ta disponível na API antiga por enquanto...
